pkg/usermanagement/client.go

```diff
@@ -347,6 +355,10 @@ type CreateOrganizationMembershipOpts struct {
 
 	// The ID of the Organization in which to add the User as a member.
 	OrganizationID string `json:"organization_id"`
+
+	// The slug of the Role in which to this membership. If not RoleSlug is given, the default role will be granted.
+	// OPTIONAL
+	RoleSlug string `json:"role_slug"`
```

Comment:
I'm guessing empty string is not a valid slug, so I think we probably want to add `omitempty` so that it isn't serialized into the request body:

```suggestion
	RoleSlug string `json:"role_slug,omitempty"`
```